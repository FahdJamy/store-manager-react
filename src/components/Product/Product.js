import React from 'react'
import { Card, Image, Button, Grid } from 'semantic-ui-react';
import "./Product.css";

const product = (props) => (
    <Card>
        <Image src='https://images.pexels.com/photos/1591057/pexels-photo-1591057.jpeg?cs=srgb&dl=buttons-characters-concept-1591057.jpg&fm=jpg' />
        <Card.Content>
            <Card.Header>{props.productName}</Card.Header>
            <Card.Meta>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's</Card.Meta>
            <Card.Description>Available stock is {props.stock}</Card.Description>
            <Card.Description>${props.productPrice}</Card.Description>
        </Card.Content>
        <Card.Content extra>
            <Grid columns={2}>
                <Grid.Column>
                    <Button color="blue" padded="very">Update</Button>
                </Grid.Column>
                <Grid.Column>
                    <Button color="red" size="medium">Delete</Button>
                </Grid.Column>
            </Grid>
        </Card.Content>
    </Card>
    
)

export default product