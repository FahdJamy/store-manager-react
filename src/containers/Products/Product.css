.ProductsLoader {
    background: #fff;
    height: 100vh;
    display: grid;
    align-items: center;
}

.AllProducts {
    height: 100vh;
}