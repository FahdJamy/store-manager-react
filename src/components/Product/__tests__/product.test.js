import React from 'react';
import { shallow } from 'enzyme';

import Product from '../Product';

describe('<Product />', () => {
    it('Product will render without crashing', () => {
        let wrapper = shallow(<Product />);
        expect(wrapper).toMatchSnapshot();
    });
});