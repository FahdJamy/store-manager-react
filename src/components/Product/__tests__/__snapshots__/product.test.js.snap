// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Product /> Product will render without crashing 1`] = `ShallowWrapper {}`;
